src/app/components/Checkbox/__tests__/Checkbox.test.tsx

```diff
@@ -1,15 +1,40 @@
 import React from "react";
 import { render } from "@testing-library/react";
-
+import { ErrorBoundary } from "react-error-boundary";
 import { Checkbox } from "../Checkbox";
 
 describe("Checkbox", () => {
+	beforeAll(() => {
+		jest.spyOn(console, "error").mockImplementation(() => null);
```

Comment:
Why was this added back?

Reply:
As I'm forcing an error in the test, it will show in the terminal. So `jest.spyOn(console, "error").mockImplementation(() => null)` hide it.

Reply:
Wouldn't that also end up swallowing legitimate errors that might be logged by some dependencies or yarn and npm itself since they use `console.*`?

Reply:
Yeah, true. I'll add:

```
afterAll(() => {
  console.error.mockRestore()
})
```

and  it works again

Reply:
That would still swallow legitimate errors while the tests run. I'll merge it for now and look at it later but these kinds of things should be avoided because they make debugging more difficult because everything in the CLI with JavaScript makes use of `console.*` to display messages and if something acts out or starts to break after a dependency update you would have to remove the mock for the logging, which isn't the first thought for most people.